.github/workflows/changelog_checker.yml

```diff
@@ -0,0 +1,19 @@
+name: "Changelog checker"
+on:
+  pull_request:
+    # The specific activity types are listed here to include "labeled" and "unlabeled"
+    # (which are not included by default for the "pull_request" trigger).
+    # This is needed to allow skipping enforcement of the changelog in PRs with specific labels,
+    # as defined in the (optional) "skipLabels" property.
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+
+jobs:
+  # Enforces the update of a changelog file on every pull request
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: dangoslen/changelog-enforcer@b2a627f5ad95e3610e22143df08656fc1a8ad130
```

Comment:
Is there a link a curious developer could follow to see a description of this action?

Reply:
Ahh, good point. I have inserted here, the same exact commit of the repository that we use at Spectro, because we security audited that one to make sure it isn't leaking code out. But it of course make the link look a little more obscure. It is just a github path, so the link is here: https://github.com/dangoslen/changelog-enforcer and the page for the GitHub action is here: https://github.com/marketplace/actions/changelog-enforcer. Should I add it in a PR or can you just do it directly on main?